bundles/binding/org.openhab.binding.lgtv/src/main/java/org/openhab/binding/lgtv/lginteraction/LgTvMessageReader.java

```diff
@@ -88,7 +88,7 @@ public LgTvMessageReader(int portno) {
     protected HttpService httpService;
 
     public void setHttpService(HttpService httpService) {
-        logger.info("sethttpservice called");
+        logger.info("setHttpService called");
```

Comment:
could we directly reduce this to debug level?

Reply:
No objections. Updated all of these.